public.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Do we know if there isn't another approach for storing/accessing keys? 

I don't feel comfortable keeping the keys in plain sight...

Reply:
apologies this shouldn't have been committed have removed the file